Allow helmfile statefile to passed by stdin (#520)
helmfile/helmfile: docs/paths.md

- Absolute paths are always resolved as absolute paths
- Relative paths referenced *in* the helmfile manifest itself are relative to that manifest
- Relative paths referenced on the command line are relative to the current working directory the user is in

> - Relative paths referenced on use `helmfile -f -` to read the yaml from the standard input are relative to the current working directory

Reviewer: ```suggestion
- Relative paths referenced from within the helmfile loaded from the standard input using `helmfile -f -` are relative to the current working directory
```
